src/mixins/createParentListMixin.ts

```diff
@@ -82,7 +77,7 @@ const createParentMixin: ParentListMixinFactory = compose<ParentList<Child>, Par
 			}
 		},
 
-		append(this: ParentListMixin<Child>, child: Child | Child[]): Handle {
+		append(this: ParentListMixin<Child>, child: (Child) | (Child)[]): Handle {
```

Comment:
I don't know what I was thinking, because it doesn't break anything when I take it out... must have been a moment of madness...  Anyways, when for `Child[] | Child`.